src/block-manager.js

```diff
@@ -102,6 +120,56 @@ Object.assign(BlockManager.prototype, require('./function-bind'), require('./med
     }
   },
 
+  getPreviousBlock: function(block) {
+    var blockPosition = this.getBlockPosition(block.el);
+    var previousBlock = this.wrapper.querySelectorAll('.st-block')[blockPosition - 1];
+    console.log(previousBlock);
+    return this.findBlockById(
+      previousBlock.getAttribute('id')
+    );
+  },
+
+  getNextBlock: function(block) {
+    var blockPosition = this.getBlockPosition(block.el);
+    return this.findBlockById(
+      this.wrapper.querySelectorAll('.st-block')[blockPosition + 1].getAttribute('id')
+    );
+  },
+
+  getBlockPosition: function(block) {
+    var index;
+    Array.prototype.forEach.call(this.wrapper.querySelectorAll('.st-block'), function(item, i) {
```

Comment:
Really we shouldn't even need to do this as blocks will be ordered in array rather than dom. I'll look at changing it to this for now.